### PR TITLE
FAQ determine meta node and auth popup updates

### PR DESCRIPTION
**Pull request by kostasb**
Documents troubleshooting steps for:

[#858](https://github.com/influxdata/docs.influxdata.com/issues/858)
Basic Auth popup thrown by the web console

[#862](https://github.com/influxdata/docs.influxdata.com/issues/862)
"Unable to determine if a remote host is a meta node"

@beckettsean @rkuchan

### CODE REVIEW

File: content/enterprise/v1.0/troubleshooting/frequently_asked_questions.md [hunk 1]
```diff
@@ -73,3 +73,17 @@ The `_internal` database collects per-node and also cluster-wide information abo
...
+
+
+## Why am I seeing `unable to determine if "<hostname>" is a meta node` when I try to add a meta node with `influxd-ctl join` ?
+
+Meta nodes use the `/status` endpoint to determine the current state of another metanode. A healthy meta node that is ready to join the cluster will respond with a `200` HTTP response code and a JSON string with the following format (assuming the default ports):
+
+`"nodeType":"meta","leader":"","httpAddr":"<hostname>:8091","raftAddr":"<hostname>:8089","peers":null}`
+
+If you are getting an error message while attempting to `influxd-ctl join` a new meta node, it means that the JSON string returned from the `/status` endpoint is incorrect. This generally indicates that the meta node configuration file is incomplete or incorrect. Inspect the HTTP response with `curl -v "http://<hostname>:8091/status"` and make sure that the `hostname`, the `bind-address`, the `http-bind-address`, and the `license-key` or `license-path` are all properly configured. Also make sure that you specify the `http-bind-address` port in the join command, e.g. `influxd-ctl join hostname:8091`.
```
Comment (beckettsean) [inline content/enterprise/v1.0/troubleshooting/frequently_asked_questions.md]: @kostasb I don't see any license info in the output. Are you directing users to fix that in the config file?

Comment (kostasb) [inline content/enterprise/v1.0/troubleshooting/frequently_asked_questions.md]: Right, this refers to the config file. Thanks for the fixes.

**Comment by beckettsean** [comment]
@rkuchan good to merge when you approve

**Comment by rkuchan** [comment]
OK! Made a tiny change to one of the questions - the markdown doesn't handle `<` well in titles. I tried `\` but it didn't help so I just removed the `<` and `>`.

Also added these changes to the 1.1 branch for Enterprise.

This is good to go! Thank you, @kostasb!

**Pull request merged**